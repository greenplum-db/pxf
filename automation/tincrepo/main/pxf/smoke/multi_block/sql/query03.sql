-- @description query03 for PXF test on Multi Blocked data

SELECT t1, a1 FROM pxf_smoke_multi_blocked_data ORDER BY t1 LIMIT 10;