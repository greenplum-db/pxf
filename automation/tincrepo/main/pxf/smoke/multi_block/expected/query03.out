-- @description query03 for PXF test on Multi Blocked data
SELECT t1, a1 FROM pxf_smoke_multi_blocked_data ORDER BY t1 LIMIT 10;
 t1 | a1 
----+----
 t1 |  1
 t1 |  1
 t1 |  1
 t1 |  1
 t1 |  1
 t1 |  1
 t1 |  1
 t1 |  1
 t1 |  1
 t1 |  1
(10 rows)

